.github/workflows/AdHocIngestion.yml

```
@@ -1,11 +1,24 @@
   contents: read
 
 on:
   workflow_dispatch:
     inputs:
+      write_mode_override:
+        description: 'Override JAVDB_HISTORY_WRITE_MODE for this run only (audit | pending). Leave empty to honour the default + .publish-config.yml override.'
+        required: false
+        default: ''
+        type: choice
+        options:
+          - ''
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)